### PR TITLE
[WIP] OeX_Depr-92 Remove edx-jsme, Molecular Structure Problem type

### PR DESCRIPTION
**Pull request by UvgenGen**
[OeX_Depr-92](https://youtrack.raccoongang.com/agiles/104-960/105-2571?issue=OeX_Depr-92)  Remove edx-jsme, Molecular Structure Problem type

https://github.com/openedx/public-engineering/issues/14

### CODE REVIEW

File: common/lib/capa/capa/responsetypes.py [hunk 1]
```diff
@@ -2094,7 +2094,7 @@ class CustomResponse(LoncapaResponse):
 
     allowed_inputfields = ['textline', 'textbox', 'crystallography',
                            'chemicalequationinput', 'vsepr_input',
-                           'drag_and_drop_input', 'editamoleculeinput',
+                           'drag_and_drop_input',
```
Comment (dyudyunov) [inline common/lib/capa/capa/responsetypes.py:2097]: Aligning items in the list will look better, but it's not necessary

File: requirements/edx-sandbox/py38.txt [hunk 2]
```diff
@@ -71,7 +71,7 @@ python-dateutil==2.4.0
     #   matplotlib
 random2==1.0.1
     # via -r requirements/edx-sandbox/py38.in
-regex==2022.3.15
+regex==2022.4.24
```
Comment (dyudyunov) [inline requirements/edx-sandbox/py38.txt:74]: Is there a particular reason for upgrading?

File: requirements/edx/base.txt [hunk 3]
```diff
@@ -396,8 +395,6 @@ djangorestframework==3.12.4
     #   edx-submissions
     #   ora2
     #   super-csv
-djangorestframework-expander==0.2.3
```
Comment (dyudyunov) [inline requirements/edx/base.txt:399]: why deleted?

File: requirements/edx/base.txt [hunk 4]
```diff
@@ -410,6 +407,8 @@ done-xblock==2.0.4
     # via -r requirements/edx/base.in
 drf-jwt==1.19.2
     # via edx-drf-extensions
+drf-nested-routers==0.93.4
```
Comment (dyudyunov) [inline requirements/edx/base.txt:410]: why added?

File: requirements/edx/coverage.txt [hunk 5]
```diff
@@ -16,5 +16,5 @@ markupsafe==2.1.1
     # via jinja2
 pluggy==1.0.0
     # via diff-cover
-pygments==2.11.2
+pygments==2.12.0
```
Comment (dyudyunov) [inline requirements/edx/coverage.txt:19]: ditto

File: requirements/edx/development.txt [hunk 6]
```diff
@@ -494,8 +495,6 @@ djangorestframework==3.12.4
     #   edx-submissions
     #   ora2
     #   super-csv
-djangorestframework-expander==0.2.3
```
Comment (dyudyunov) [inline requirements/edx/development.txt:497]: ditto

File: requirements/edx/development.txt [hunk 7]
```diff
@@ -517,6 +516,10 @@ drf-jwt==1.19.2
     # via
     #   -r requirements/edx/testing.txt
     #   edx-drf-extensions
+drf-nested-routers==0.93.4
```
Comment (dyudyunov) [inline requirements/edx/development.txt:519]: ditto

File: requirements/edx/development.txt [hunk 8]
```diff
@@ -1067,7 +1074,7 @@ pydantic==1.9.0
     # via
     #   -r requirements/edx/testing.txt
     #   fastapi
-pygments==2.11.2
+pygments==2.12.0
```
Comment (dyudyunov) [inline requirements/edx/development.txt:1077]: ditto

File: requirements/edx/development.txt [hunk 9]
```diff
@@ -1115,7 +1122,7 @@ pylint-plugin-utils==0.7
     #   pylint-django
 pylint-pytest==0.3.0
     # via -r requirements/edx/testing.txt
-pylti1p3==1.10.0
+pylti1p3==1.11.0
```
Comment (dyudyunov) [inline requirements/edx/development.txt:1125]: ditto

File: requirements/edx/development.txt [hunk 10]
```diff
@@ -1143,7 +1150,7 @@ pysrt==1.1.2
     # via
     #   -r requirements/edx/testing.txt
     #   edxval
-pytest==7.1.1
+pytest==7.1.2
```
Comment (dyudyunov) [inline requirements/edx/development.txt:1153]: ditto

File: requirements/edx/development.txt [hunk 11]
```diff
@@ -1251,7 +1258,7 @@ recommender-xblock==2.0.1
     # via -r requirements/edx/testing.txt
 redis==4.2.2
     # via -r requirements/edx/testing.txt
-regex==2022.3.15
+regex==2022.4.24
```
Comment (dyudyunov) [inline requirements/edx/development.txt:1261]: ditto

File: requirements/edx/base.txt [hunk 12]
```diff
@@ -786,7 +789,7 @@ pycryptodomex==3.14.1
     #   edx-proctoring
     #   lti-consumer-xblock
     #   pyjwkest
-pygments==2.11.2
+pygments==2.12.0
```
Comment (dyudyunov) [inline requirements/edx/base.txt:792]: why upgraded?

File: requirements/edx/base.txt [hunk 13]
```diff
@@ -807,7 +810,7 @@ pyjwt[crypto]==2.3.0
     #   social-auth-core
 pylatexenc==2.10
     # via olxcleaner
-pylti1p3==1.10.0
+pylti1p3==1.11.0
```
Comment (dyudyunov) [inline requirements/edx/base.txt:813]: ditto

File: requirements/edx/base.txt [hunk 14]
```diff
@@ -898,7 +901,7 @@ recommender-xblock==2.0.1
     # via -r requirements/edx/base.in
 redis==4.2.2
     # via -r requirements/edx/base.in
-regex==2022.3.15
+regex==2022.4.24
```
Comment (dyudyunov) [inline requirements/edx/base.txt:904]: ditto

File: requirements/edx/doc.txt [hunk 15]
```diff
@@ -44,7 +44,7 @@ packaging==21.3
     # via sphinx
 pbr==5.8.1
     # via stevedore
-pygments==2.11.2
+pygments==2.12.0
```
Comment (dyudyunov) [inline requirements/edx/doc.txt:47]: ditto

File: requirements/edx/testing.txt [hunk 16]
```diff
@@ -481,10 +480,6 @@ djangorestframework==3.12.4
     #   edx-submissions
     #   ora2
     #   super-csv
-djangorestframework-expander==0.2.3
```
Comment (dyudyunov) [inline requirements/edx/testing.txt:484]: ditto

File: requirements/edx/testing.txt [hunk 17]
```diff
@@ -504,6 +499,10 @@ drf-jwt==1.19.2
     # via
     #   -r requirements/edx/base.txt
     #   edx-drf-extensions
+drf-nested-routers==0.93.4
```
Comment (dyudyunov) [inline requirements/edx/testing.txt:502]: ditto

File: requirements/edx/testing.txt [hunk 18]
```diff
@@ -662,7 +662,7 @@ execnet==1.9.0
     # via pytest-xdist
 factory-boy==3.2.1
     # via -r requirements/edx/testing.in
-faker==13.3.5
```
Comment (dyudyunov) [inline requirements/edx/testing.txt:665]: ditto

File: requirements/edx/testing.txt [hunk 19]
```diff
@@ -1010,7 +1011,7 @@ pycryptodomex==3.14.1
     #   pyjwkest
 pydantic==1.9.0
     # via fastapi
-pygments==2.11.2
```
Comment (dyudyunov) [inline requirements/edx/testing.txt:1013]: ditto

File: requirements/edx/testing.txt [hunk 20]
```diff
@@ -1052,7 +1053,7 @@ pylint-plugin-utils==0.7
     #   pylint-django
 pylint-pytest==0.3.0
     # via -r requirements/edx/testing.in
-pylti1p3==1.10.0
```
Comment (dyudyunov) [inline requirements/edx/testing.txt:1055]: ditto

File: requirements/edx/testing.txt [hunk 21]
```diff
@@ -1078,7 +1079,7 @@ pysrt==1.1.2
     # via
     #   -r requirements/edx/base.txt
     #   edxval
-pytest==7.1.1
+pytest==7.1.2
```
Comment (dyudyunov) [inline requirements/edx/testing.txt:1082]: ditto

File: requirements/edx/testing.txt [hunk 22]
```diff
@@ -1181,7 +1182,7 @@ recommender-xblock==2.0.1
     # via -r requirements/edx/base.txt
 redis==4.2.2
     # via -r requirements/edx/base.txt
-regex==2022.3.15
+regex==2022.4.24
```
Comment (dyudyunov) [inline requirements/edx/testing.txt:1185]: ditto

**Review by dyudyunov — COMMENTED** [review]
Why there are so many changes in the requirements? Do all of them relate to edx-jsme deletion?

**Review by dyudyunov — APPROVED** [review]

**Comment by dyudyunov** [comment]
👍

**Review by idegtiarov — APPROVED** [review]
LGTM!

**Pull request closed without merging**